bin/cml-runner.js

```diff
@@ -394,6 +396,8 @@ const opts = yargs
     'cloud-startup-script',
     'Run the provided Base64-encoded Linux shell script during the instance initialization'
   )
+  .default('cloud-aws-security-group', '')
```

Comment:
minor note: should we use `.default('thing')` or `.default('thing', '')`? Both seem to be used in this file.

Reply:
Good question! opens #592